### ISSUE TITLE
Missing data package/folder from pytorch_forecast package

### ISSUE THREAD

**Issue by PlamenStilyianov** [PacktPublishing/Modern-Time-Series-Forecasting-with-Python-2E#35]
When you run the following script  test_installation.py file, those packages fail

from pytorch_forecasting import TimeSeriesDataSet
from pytorch_forecasting.data import GroupNormalizer

synchronyse it with manu's repository
https://github.com/manujosephv/pytorch-forecasting/tree/master/pytorch_forecasting

the packages it looks like will be

from pytorch_forecasting.timeseries import TimeSeriesDataSet
from pytorch_forecasting.data.encoders import GroupNormalizer

**Comment by PlamenStilyianov** [comment]
from pytorch_forecasting.data.timeseries import TimeSeriesDataSet

**Comment by PlamenStilyianov** [comment]
The requirements_new.txt file needs to add
pytorch-forecasting>=1.2.0

and to rename pytorch_forecasting in the structure as pytorch_forecasting_redundant

and to change the test_installation.py as above

**Comment by manujosephv** [comment]
I guess you might not be running the codes from the root folder of the repo? We have included the pytorch_forecasting codes directly in the root of the directory so that it can be loaded. But if you have an installation of pytorch_forecasting already in your environment, it might be taking that.

Also, adding pytorch_forecasting>=1.2.0 isn't a great idea. `pytorch_forecasting` isn't maintained and still requires an older version of `pytroch_lightning` and that conflicts with `neuralforecast` which requires newer version of `pytroch_lightning`. That is why I included the code from pytorch_forecasting directly in the repo.

**Comment by bulat-khusnimardanov** [comment]
> I guess you might not be running the codes from the root folder of the repo? We have included the pytorch_forecasting codes directly in the root of the directory so that it can be loaded. But if you have an installation of pytorch_forecasting already in your environment, it might be taking that.
> 
> Also, adding pytorch_forecasting>=1.2.0 isn't a great idea. `pytorch_forecasting` isn't maintained and still requires an older version of `pytroch_lightning` and that conflicts with `neuralforecast` which requires newer version of `pytroch_lightning`. That is why I included the code from pytorch_forecasting directly in the repo.

Hi @manujosephv,

I encountered the same issue while running the code from the root folder of the repository. The `pytorch_forecasting` directory is present, but it appears that the `pytorch_forecasting/data` subdirectory is missing. As a result, the `test_installation.py` script fails with the following error:
`ModuleNotFoundError: No module named 'pytorch_forecasting.data'`

**Comment by manujosephv** [comment]
Yeah. I just checked that out. The folder is missing. turns out the exception to `.gitignore` I added to not check in data files also didn't check in the data folder in `pytorch_forecasting`  😁 

I've checked it in now.

**Comment by bulat-khusnimardanov** [comment]
Thanks a lot! It's working perfectly now.
`######################### All Libraries imported without errors! #########################`
